src/main/java/org/compi/csc311group3/database/DbConnection.java

```diff
@@ -37,4 +37,37 @@ password VARCHAR(200)
 
         return conn;
     }
+    // Method to initialize the expense table
+    /**+----+--------+-------------+-------------------------+---------------------+---------+
+     | id | amount | type        | description             | expense_date        | user_id |
+     +----+--------+-------------+-------------------------+---------------------+---------+
+     |  1 |  25.50 | Food        | Lunch at a restaurant   | 2024-11-20 12:34:56 |       1 |
+     |  2 |  15.00 | Transport   | Bus ticket              | 2024-11-19 08:22:10 |       2 |
+     |  3 | 100.00 | Utilities   | Electricity bill        | 2024-11-18 17:00:00 |       1 |
+     |  4 |  50.00 | Entertainment | Movie tickets          | 2024-11-20 19:45:00 |       3 |
+     |  5 |  75.00 | Food        | Grocery shopping        | 2024-11-17 14:30:45 |       2 |
+     +----+--------+-------------+-------------------------+---------------------+---------+
+*/
+     public void initializeExpenseTable() {
```

Comment:
I believe this should be within the getConnection() method. Perhaps we create Create[name} methods for each table, that we call inside getConnection. Do you have anything against this? I can implement it quickly and commit into this PR if you'd like me to. It would negate the need to call initializeDatabase in other classes.

Reply:
yea go ahead, i dont mind at all